salt/modules/nexus.py

```diff
@@ -466,13 +467,15 @@ def __save_artifact(artifact_url, target_file, headers):
             result['comment'] = checksum_comment
             return result
 
-    log.debug('Downloading: {url} -> {target_file}'.format(url=artifact_url, target_file=target_file))
+    log.debug('Downloading: %s -> %s', url=artifact_url, target_file=target_file)
```

Comment:
The `url=` and `target_file=` need to be removed, they were only there as part of the `.format()`.